spec/system/consumer/split_checkout_spec.rb

```diff
@@ -128,16 +128,20 @@
           end
 
           it "redirects the user to the Payment Method step, when submiting the form" do
-            click_button "Next - Payment method"
-            expect(page).to have_current_path("/checkout/payment")
+            proceed_to_payment
+            # asserts whether shipping and billing addresses are the same
+            ship_add_id = Spree::Order.first.ship_address_id
+            bill_add_id = Spree::Order.first.bill_address_id
```

Comment:
Why not `order.reload.ship_address_id`? Is it not the same?

Reply:
You need to reload the order object only once to get the updates from the database but your version looks better. :wink: